SAI/README.md

```diff
@@ -0,0 +1,21 @@
+```
+usage: sai_api_gen.py [-h] [--print-sai-lib PRINT_SAI_LIB]
```

Comment:
I tried it twice in a row and got errors:
```
SAI$ ./sai_api_gen.py ../sirius-pipeline/bmv2/sirius_pipeline.bmv2/sirius_pipeline.json dash
Generating SAI API...
Cloning SAI repository...
#
# Repeat
#
SAI$ ./sai_api_gen.py ../sirius-pipeline/bmv2/sirius_pipeline.bmv2/sirius_pipeline.json dash
Generating SAI API...
Cloning SAI repository...
Traceback (most recent call last):
  File "./sai_api_gen.py", line 235, in <module>
    Repo.clone_from(args.sai_git_url, './SAI', branch=args.sai_git_branch)
  File "/home/chris/.local/lib/python3.6/site-packages/git/repo/base.py", line 1111, in clone_from
    return cls._clone(git, url, to_path, GitCmdObjectDB, progress, multi_options, **kwargs)
  File "/home/chris/.local/lib/python3.6/site-packages/git/repo/base.py", line 1049, in _clone
    finalize_process(proc, stderr=stderr)
  File "/home/chris/.local/lib/python3.6/site-packages/git/util.py", line 370, in finalize_process
    proc.wait(**kwargs)
  File "/home/chris/.local/lib/python3.6/site-packages/git/cmd.py", line 447, in wait
    raise GitCommandError(remove_password_if_present(self.args), status, errstr)
git.exc.GitCommandError: Cmd('git') failed due to: exit code(128)
  cmdline: git clone -v --branch=master https://github.com/Opencomputeproject/SAI ./SAI
  stderr: 'fatal: destination path './SAI' already exists and is not an empty directory.
'
```
Script should not try to clone repo if `SAI` directory already exists.

Reply:
Added error handling for the case when directory already exists.